mailbox/event/json/src/main/scala/org/apache/james/event/json/DTOs.scala

```diff
@@ -57,7 +55,7 @@ object DTOs {
 
   case class ACLDiff(oldACL: Map[MailboxACL.EntryKey, MailboxACL.Rfc4314Rights],
                      newACL: Map[MailboxACL.EntryKey, MailboxACL.Rfc4314Rights]) {
-    def toJava: JavaACLDiff = new JavaACLDiff(new MailboxACL(oldACL.asJava), new MailboxACL(newACL.asJava))
+    def toJava: JavaACLDiff = JavaACLDiff.computeDiff(new MailboxACL(oldACL.asJava), new MailboxACL(newACL.asJava))
```

Comment:
JavaACLDiff is not longer needed as ACLDiff is scala. No facade is any longer required. If you wonna save line of code, look in that direction

Reply:
I know. I also don't want to change too many things at the same time.